test/component/coapplicant/testCoApplicantAllAgreePage.js

```diff
@@ -24,27 +24,21 @@ describe('co-applicant-all-agreed-page', () => {
     describe('Verify Content, Errors and Redirection', () => {
         it('test correct content is loaded on the page when there are no codicils', (done) => {
             sessionData.formdata.will.codicils = commonContent.no;
-            const contentToExclude = [
-                'paragraph4-codicils'
-            ];
             checkAllAgreedStub.returns(Promise.resolve('true'));
             testWrapper.agent.post('/prepare-session/form')
                 .send(sessionData.formdata)
                 .end(() => {
-                    testWrapper.testContent(done, contentToExclude, contentData);
+                    testWrapper.testContent(done, [], contentData);
                 });
         });
 
         it('test correct content is loaded on the page when there are codicils', (done) => {
```

Comment:
We don't distinguish between codicils and no codicils in the content so we can remove this test